pom.xml

```diff
@@ -431,9 +431,10 @@
           <failOnWarning>false</failOnWarning>
           <sourceDirectory>${basedir}/src/main/scala</sourceDirectory>
           <testSourceDirectory>${basedir}/src/test/scala</testSourceDirectory>
-          <configLocation>${dev.path}/scalastyle-config.xml</configLocation>
+          <configLocation>scalastyle-config.xml</configLocation>
```

Comment:
why. do you remove ${dev.path}/?

Reply:
moved scalastyle-config.xml into the parent project folder.